src/callbacks_step/analysis_surface_2d.jl

```diff
@@ -0,0 +1,138 @@
+    # Ψd is the unit vector parallel to the freestream direction
+    Ψd = (cos(aoa), sin(aoa))
+    return DragCoefficient(ForceState(Ψd, rhoinf, uinf, linf))
+end
+
+function (lift_coefficient::LiftCoefficient)(u, normal_direction, equations)
+    p = pressure(u, equations)
+    @unpack Ψ, rhoinf, uinf, linf = lift_coefficient.force_state
+    n = dot(normal_direction, Ψ) / norm(normal_direction)
+    return p * n / (0.5 * rhoinf * uinf^2 * linf)
+end
+
+function (drag_coefficient::DragCoefficient)(u, normal_direction, equations)
+    p = pressure(u, equations)
+    @unpack Ψ, rhoinf, uinf, linf = drag_coefficient.force_state
+    n = dot(normal_direction, Ψ) / norm(normal_direction)
+    return p * n / (0.5 * rhoinf * uinf^2 * linf)
+end
+
+function analyze(surface_variable::AnalysisSurfaceIntegral, du, u, t,
+                 mesh::P4estMesh{2},
+                 equations, dg::DGSEM, cache)
+    @unpack boundaries = cache
+    @unpack surface_flux_values, node_coordinates, contravariant_vectors = cache.elements
+    @unpack weights = dg.basis
+    @unpack semi, indices, variable = surface_variable
+    indices_ = indices(semi)
+
+    surface_integral = zero(eltype(u))
+    index_range = eachnode(dg)
+    for local_index in eachindex(indices_)
+        # Use the local index to get the global boundary index from the pre-sorted list
+        boundary = indices_[local_index]
+
+        # Get information on the adjacent element, compute the surface fluxes,
+        # and store them
+        element = boundaries.neighbor_ids[boundary]
+        node_indices = boundaries.node_indices[boundary]
+        direction = indices2direction(node_indices)
+
+        i_node_start, i_node_step = index_to_start_step_2d(node_indices[1], index_range)
+        j_node_start, j_node_step = index_to_start_step_2d(node_indices[2], index_range)
+
+        i_node = i_node_start
+        j_node = j_node_start
+        for node_index in eachnode(dg)
+            u_node = Trixi.get_node_vars(cache.boundaries.u, equations, dg, node_index,
+                                         boundary)
+            normal_direction = get_normal_direction(direction, contravariant_vectors,
+                                                    i_node, j_node,
+                                                    element)
+
+            # L2 norm of normal direction is the surface element
+            dS = weights[node_index] * norm(normal_direction)
+            surface_integral += variable(u_node, normal_direction, equations) * dS
+
+            i_node += i_node_step
+            j_node += j_node_step
+        end
+    end
+    return surface_integral
+end
+
+function pretty_form_ascii(::AnalysisSurfaceIntegral{<:Any, <:Any,
+                                                     <:LiftCoefficient{<:Any}})
+    "CL"
+end
+function pretty_form_utf(::AnalysisSurfaceIntegral{<:Any, <:Any,
+                                                   <:LiftCoefficient{<:Any}})
+    "CL"
```

Comment:
Is it possible to use for `utf` actually the subscript, i.e., $C_L$ ?

Reply:
You can find a list of superscript/subscript indices, e.g., [here](https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts#Superscripts_and_subscripts_block).

Reply:
Unicode characters only give us Cₗ (small 'l') and nothing for the 'D'.

I verified it from the link shared by @sloede and some subscript generators.

Reply:
Hm yeah just saw that. Fortunately we have :joy_cat: which we could use instead ...

Reply:
Could you explain what you mean? Do you have a particular emoticon in mind?

Reply:
I was not serious :D Actually you can code in Julia using those emojis but cannot use every letter as a subscript - I was mocking this a bit.

Reply:
Ah, that is so obvious in retrospect!